### PR TITLE
Upgrade to latest OKHttp client and related bundle 3.7.0

### PR DESCRIPTION
**Pull request by oscerd**
Now that we have the 3.7.0 bundle too we can upgrade.

### CODE REVIEW

**Comment by oscerd** [comment]
@jimmidyson @iocanel

File: pom.xml [hunk 1]
```diff
@@ -75,8 +75,8 @@
   <properties>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <mockwebserver.version>0.0.12</mockwebserver.version>
-    <okhttp.version>3.6.0</okhttp.version>
-    <okhttp.bundle.version>3.6.0_1</okhttp.bundle.version>
+    <okhttp.version>3.7.0</okhttp.version>
+    <okhttp.bundle.version>3.7.0_1</okhttp.bundle.version>
     <okio.version>1.11.0</okio.version>
```
Comment (jimmidyson) [inline pom.xml]: This needs to be bumped to 1.12.0 I think.

**Comment by jimmidyson** [comment]
Thanks! Should be a drop in replacement, but can you please bump okio too?

**Comment by oscerd** [comment]
Sure, I'm running a full build locally with okio 1.12.0

**Comment by oscerd** [comment]
Done :-)

**Comment by jimmidyson** [comment]
Thanks @oscerd! [merge]

**Pull request merged**